### ISSUE TITLE
Filename completion should be a function of history

### ISSUE THREAD

**Issue by jimt** [warpdotdev/warp#273]
### Describe the solution you'd like?

```shell
If I do `nvim r<TAB>`, the default completion should be either:

* the last one I selected
* the filename I selected most often

I would prefer the first, but can imagine some preferring the second.
```

**Comment by elviskahoro** [comment]
@jimt I'm personally interested in your suggestion. A couple of clarification questions:
- Is this in the context of fzf searching through absolute paths?
- Is it suggesting filenames in the current directory that start with r, but the prioritized / first suggestion is (the most recently selected filename)?

**Comment by jimt** [comment]
Yes.  I was thinking mostly of the second, but in the more rare times when I use the absolute path, I suppose I expect the former.

But it is not independent. For example, if I am using a tool, I want the files that I have either most recently or most often used as arguments for that particular tool.

**Comment by elviskahoro** [comment]
We have an engineer who's gonna be working on this over the quarter and so expect improvements!
